File: geoquiz/src/main/java/com/fenfei/geoquiz/MainActivity.java
package com.fenfei.geoquiz;

import android.app.ActivityManager;
import android.content.Context;
import android.os.Build;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;
import android.view.ViewAnimationUtils;

public class MainActivity extends AppCompatActivity {

    private String TAG = "MainActivity";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ViewAnimationUtils animationUtils;

        ActivityManager activityManager = (ActivityManager) getSystemService(Context.ACTIVITY_SERVICE);

        int SDK_INT = Build.VERSION.SDK_INT;
        Log.e(TAG, "onCreate: " + SDK_INT );

    }
}

File: dynamticbuild/src/demo/java/com/fenfei/myapplication/demo/MainActivity.kt
package com.fenfei.myapplication.demo

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log
import com.fenfei.myapplication.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Log.e("demo" , "Demo starting")
    }



}
